docs/BreakingChanges/055 Unobserved exceptions in System.Threading.Tasks.Task operations.md

```diff
@@ -0,0 +1,31 @@
+## 55: Unobserved exceptions in System.Threading.Tasks.Task operations
```

Comment:
{
  "Id": "55",
  "Title": "Unobserved exceptions in System.Threading.Tasks.Task operations",
  "Details": "Because the System.Threading.Tasks.Task class represents an asynchronous operation, it catches all non-severe exceptions that occur during asynchronous processing. In the .NET Framework 4.5, if an exception is not observed and your code never waits on the task, the exception will no longer propagate on the finalizer thread and crash the process during garbage collection. This change enhances the reliability of applications that use the Task class to perform unobserved asynchronous processing.",
  "Suggestion": "If an app depends on unobserved asynchronous exceptions propagating to the finalizer thread, the previous behavior can be restored by providing an appropriate handler for the [TaskScheduler.UnobservedTaskException](https://msdn.microsoft.com/en-us/library/system.threading.tasks.taskscheduler.unobservedtaskexception(v=vs.110).aspx) event.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#core",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Threading.Tasks.Task.Run(System.Action)",
    "M:System.Threading.Tasks.Task.Run(System.Action,System.Threading.CancellationToken)",
    "M:System.Threading.Tasks.Task.Run(System.Func{System.Threading.Tasks.Task})",
    "M:System.Threading.Tasks.Task.Run(System.Func{System.Threading.Tasks.Task},System.Threading.CancellationToken)",
    "M:System.Threading.Tasks.Task.Run`1(System.Func{`0})",
    "M:System.Threading.Tasks.Task.Run`1(System.Func{`0},System.Threading.CancellationToken)",
    "M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{`0}})",
    "M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{`0}},System.Threading.CancellationToken)",
    "M:System.Threading.Tasks.Task.Start",
    "M:System.Threading.Tasks.Task.Start(System.Threading.Tasks.TaskScheduler)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Threading.Tasks.Task.Run(System.Action)
- [x] M:System.Threading.Tasks.Task.Run(System.Action,System.Threading.CancellationToken)
- [x] M:System.Threading.Tasks.Task.Run(System.Func{System.Threading.Tasks.Task})
- [x] M:System.Threading.Tasks.Task.Run(System.Func{System.Threading.Tasks.Task},System.Threading.CancellationToken)
- [ ] M:System.Threading.Tasks.Task.Run`1(System.Func{`0})
- [ ] M:System.Threading.Tasks.Task.Run`1(System.Func{`0},System.Threading.CancellationToken)
- [ ] M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{`0}})
- [ ] M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{`0}},System.Threading.CancellationToken)
- [x] M:System.Threading.Tasks.Task.Start
- [x] M:System.Threading.Tasks.Task.Start(System.Threading.Tasks.TaskScheduler)